packages/docusaurus-theme-classic/codeTranslations/base.json

```diff
@@ -10,7 +10,9 @@
   "theme.blog.post.paginator.newerPost": "Newer Post",
   "theme.blog.post.paginator.olderPost": "Older Post",
   "theme.blog.post.readMore": "Read More",
-  "theme.blog.tags": "Tags:",
+  "theme.tags.tagsPageLink": "View All Tags",
+  "theme.tags.tagsPageTitle": "Tags",
+  "theme.tags.tagsListLabel": "Tags:",
```

Comment:
Looks like duplicate to me, colon should not be a apart of translation string which will allow to reuse the 18n string.